src/components/Header/Header.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { NavLink, Link } from 'react-router-dom';
+import { AppBar, Box, Toolbar, Typography, Button } from '@mui/material';
+
+const Header = () => {
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar position="static">
+        <Toolbar>
+          <Link to="/" style={{ textDecoration: 'none', color: 'white' }}>
+            <Typography
+              align="left"
+              variant="h6"
+              component="div"
+              sx={{ textDecoration: 'none' }}
+            >
+              Rick and Morty Characters App
+            </Typography>
+          </Link>
```

Comment:
I'd create a `NavOption` component so this list will be very clear and easy on the eyes